src/components/v5/common/ActionSidebar/partials/forms/SplitPaymentForm/hooks.ts

```diff
@@ -0,0 +1,130 @@
+import * as yup from 'yup';
+import { useCallback, useMemo } from 'react';
+import { Id } from '@colony/colony-js';
+import { useWatch } from 'react-hook-form';
+import { ActionTypes } from '~redux';
+import { mapPayload, pipe } from '~utils/actions';
+import { useColonyContext, useNetworkInverseFee } from '~hooks';
+import { getCreatePaymentDialogPayload } from '~common/Dialogs/CreatePaymentDialog/helpers';
+import { MAX_ANNOTATION_NUM } from '~v5/shared/RichText/consts';
+import { toFinite } from '~utils/lodash';
+import { ActionFormBaseProps } from '../../../types';
+import { useActionFormBaseHook } from '../../../hooks';
+import { DECISION_METHOD_OPTIONS } from '../../consts';
+import { notNull } from '~utils/arrays';
+
+const validationSchema = yup
+  .object({
+    amount: yup
+      .object({
+        amount: yup
+          .number()
+          .required(() => 'required field')
+          .transform((value) => toFinite(value))
+          .moreThan(0, () => 'Amount must be greater than zero'),
+        tokenAddress: yup.string().address().required(),
+      })
+      .required(),
+    createdIn: yup.string().defined(),
+    description: yup.string().max(MAX_ANNOTATION_NUM).notRequired(),
+    team: yup.string().required(),
+    decisionMethod: yup.string().defined(),
+    distributionMethod: yup.string().defined(),
+    payments: yup
+      .array(
+        yup.object().shape({
+          percent: yup.number().required(),
+          recipient: yup.string().required(),
+        }),
+      )
+      .test('sum', 'The sum of percentages must be 100', (value) => {
+        if (!value) {
+          return false;
+        }
+
```

Comment:
remove empty space

Reply:
I usually separate `if` blocks and `return` statements with new lines. Maybe I am not aware of some coding standards in this project, can you point me to some docs regarding separating code blocks?

Reply:
ok